.editorconfig

```diff
@@ -17,3 +17,16 @@ dotnet_sort_system_directives_first = true
 
 dotnet_diagnostic.IDE0073.severity = warning  # Require file header
 file_header_template = Copyright (c) Duende Software. All rights reserved.\nSee LICENSE in the project root for license information.
+
+dotnet_diagnostic.IDE0161.severity = error  # Use file-scoped namespace
+csharp_style_namespace_declarations = file_scoped:error
+
+dotnet_diagnostic.IDE0003.severity = error # Enforce that 'this.' should not be used when it's redundant
+
+dotnet_diagnostic.IDE0049.severity = error # Enforce using built-in type keywords (e.g., 'string' instead of 'String')
```

Comment:
Title is "Use language keywords instead of framework type names for type references"

Reply:
https://github.com/DuendeSoftware/products/pull/1877/files